README.md

```diff
@@ -8,18 +8,22 @@ A curated list of awesome [streaming (stream processing)](http://radar.oreilly.c
 
 ## Table of Contents
 
-- [Streaming Engine](#streaming-engine)
-- [Streaming Library](#streaming-library)
-- [Streaming Application](#streaming-application)
-- [IoT](#iot)
-- [DSL](#dsl)
-- [Data Pipeline](#data-pipeline)
-- [Online Machine Learning](#online-machine-learning)
-- [Streaming SQL](#streaming-sql)
-- [Toolkit](#toolkit)
-- [Benchmark](#benchmark)
-- [Closed Source](#closed-source)
-- [Readings](#readings)
+- [Awesome Streaming   ](#awesome-streaming---)
```

Comment:
please don't change these lines.

Reply:
Yes, I fixed it now. It was autocomplete by VS code:)